homeassistant/components/dnsip/sensor.py

```diff
@@ -75,12 +82,13 @@ def __init__(
         hostname: str,
         resolver: str,
         ipv6: bool,
+        port: int = 53,
```

Comment:
```suggestion
        port: int,
```
No need as we have a value always